internal/provisioner/kops_provisioner_cluster.go

```diff
@@ -367,6 +364,20 @@ func (provisioner *KopsProvisioner) ProvisionCluster(cluster *model.Cluster, aws
 		},
 	}
 
+	if len(cluster.ProvisionerMetadataKops.Networking) > 0 && (cluster.ProvisionerMetadataKops.Networking == "amazonvpc" || cluster.ProvisionerMetadataKops.Networking == "weave") {
```

Comment:
I think we may also want to check `cluster.ProvisionerMetadataKops != nil` otherwise in some edge cases checking `cluster.ProvisionerMetadataKops.Networking' on nill parent may cause nil pointer exception